composer.json

```diff
@@ -1,5 +1,5 @@
 {
-    "name": "jimdo/prometheus_client_php",
+    "name": "imj/prometheus_client_php",
```

Comment:
You could always instruct composer to use your repository, without needing to change the package name. See https://getcomposer.org/doc/05-repositories.md#loading-a-package-from-a-vcs-repository